src/pygeosx/unitTests/pytest_template.sh.in

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+PYTHONPATH=${CMAKE_BINARY_DIR}/lib/PYGEOSX/lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/:$PYTHONPATH ${Python3_EXECUTABLE} -m pytest ${CMAKE_CURRENT_SOURCE_DIR}/${filename}::test_${name}
```

Comment:
I'm OK with this for the moment, but I'm not sure about having a unit test fwk (`ctest`) on top of another unit test fwk (`pytest`).

Reply:
@TotoGaz is pytest run today by default on CI configurations?